### DISCUSSION TITLE
When running Git Bash prompt as a non-admin, the following error is always reported

### DISCUSSION THREAD

**Discussion by jetjo** [git-for-windows/git#3271] [Q&A]
### Current behavior:
When I run `git status` in a git repository,  Or ,After I did some coding work and saved it,then run `git add .`,the following error is reported:
`error: cannot spawn git: Function not implemented`
If I first run Git Bash prompt as Administrator, and then start another one as Non-Administrator, and run the above command in this Non-Administrator terminal, another error is reported:
`error: fsmonitor_ipc__send_query: unspecified error on '.git/fsmonitor--daemon.ipc'`
I found that if you don't start the Git Bash prompt once as Administrator, there is no `git.exe` process in the Task Manager,the Command Line info is:
`"C:\Program Files\Git\mingw64\libexec\git-core\git.exe" fsmonitor--daemon run`
### OS Versions:
Windows 10 Build 19043.1052 (21H1)
### Git Bash Versions:
[https://github.com/git-for-windows/git/releases/tag/v2.32.0.windows.1](url)
### Other
it is very too much annoying that runnig a program with Administrator permissions and UAC prompt every time!!!

**Comment by dscho** [comment]
It sounds as if this might be related to https://github.com/git-for-windows/git/issues/2710.
